scripts/gitlab/hatchet-analysis.py

```diff
@@ -22,32 +23,100 @@
 
 import hatchet as ht
 
-f1 = options.report
-f2 = options.baseline
-
-gf1 = ht.GraphFrame.from_caliperreader(f1)
-gf2 = ht.GraphFrame.from_caliperreader(f2)
-
-gf3 = gf2 - gf1
-
-# Display dataframe columns
-print(gf3.dataframe.columns)
+class GenericFrame(ht.GraphFrame):
+   def __init__(self, gf):
+      generic_dataframe = gf.dataframe.copy()
+      generic_graph = gf.graph.copy()
+      generic_exc_metrics = gf.exc_metrics
+      generic_inc_metrics = gf.inc_metrics
```

Comment:
Those metrics are new class variables which are copied over from graphframe argument,  and eventually used as arguments to initialize the super class super().__init__, you can see that this initialization is all copies, sometime explicit as in generic_dataframe=gf.dataframe.copy(). Which allows me to create a new graphframe in this type of call

gf1 = GenericFrame(ht.GraphFrame.from_caliperreader(f1))

what GenericFrame is doing is renaming the root node to name "Variant" and setting all the attributes that makes it a "Node". This will allow Hatchet to compare completely different trees (since the root node itself is different), while assuming that the tree structure underneath in the two trees are completely identical. If they are not identical, then we extractCommonSubtree so we can compare apples to apples